dali/kernels/reduce/reduce_cpu.h

```diff
@@ -197,7 +197,7 @@ struct ReduceBaseCPU {
     auto R = This().GetReduction();
     if (axis == output.dim()) {
       Dst &r = *output(pos);
-      if (clear) r = 0;
+      if (clear) r = R.template neutral<Dst>();
```

Comment:
Nice catch :+1: 

Nitpick:
```suggestion
      if (clear)
        r = R.template neutral<Dst>();
```

Reply:
Done